### DISCUSSION TITLE
[4.2x] Feature request: allow linking of intro images on a per-article basis in category blog

### DISCUSSION THREAD

**Discussion by Scrabble96** [joomla/joomla-cms#47095] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
Not a problem, but...

Create a blog with intro images. Some articles are long and have a 'read more' link added. Some articles are short and don't need a 'read more' link. There is the option to turn off linked titles in the individual articles, but only a global setting for linked intro images in the category blog menu item setup (under the 'Blog Layout' tab). If this option is on, then anyone clicking an image - as they might be tempted to do, because the mouse pointer changes to a hand at this point - will be disappointed not to see more text.

### Describe the solution you'd like
Add an option to link intro images per article in the article options.

**Comment by brianteeman** [comment]
actually its a bug. a link should not be created if there is no readmore but there is no check for this in the layout

https://github.com/joomla/joomla-cms/blob/766226e7f6d8ffde9427905f18158e2bd7d5bf26/layouts/joomla/content/intro_image.php#L31-L34

**Comment by carlitorweb** [comment]
@brianteeman then the link in the title cant be created too if there is not readmore, right? I think, the readmore button should not decide if the image or the title will have a link too. Maybe hidden the readmore button is just part of the page style/desing, and not as defining the article will not have a link at all. What do you think?

**Comment by Hackwar** [comment]
Not a bug, but a request for changed behavior. Marking thus as feature for now.
